### PR TITLE
fix(password): Add password reset button

### PR DESCRIPTION
**Pull request by mindreeper2420**
Add a button for resetting a user's password from their profile.

related to https://github.com/fabric8-ui/fabric8-ux/issues/801

**Note:** This is only the HTML/CSS for adding the password reset button. This does not include the linkage to activate this functionality.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@mindreeper2420 fabric8/fabric8-ui:SNAPSHOT-PR-2462-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2462-fabric8-ui.badger.fabric8.io

File: src/app/profile/update/update.component.html [hunk 1]
```diff
@@ -82,6 +82,11 @@ <h1>Edit Profile</h1>
             <span class="fields-status-pf">characters remaining</span>
           </span>
         </div>
+        <div class="form-group">
+          <button class="btn btn-default" id="passwordReset">
+            Change password <i class="fa fa-external-link"></i>
```
Comment (alexeykazakov) [inline src/app/profile/update/update.component.html]: This is the link we should use here - https://developers.redhat.com/auth/realms/rhd/account/password
Could you update your PR with that link?

Comment (mindreeper2420) [reply src/app/profile/update/update.component.html]: Added - tested locally and it brought me to the proper password reset page.

**Comment by fabric8cd** [comment]
@mindreeper2420 fabric8/fabric8-ui:SNAPSHOT-PR-2462-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2462-fabric8-ui.badger.fabric8.io

**Pull request merged**